hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -3216,6 +3216,24 @@ public static <T> String safeGetAsStr(List<T> lst, int i) {
     }
   }
 
+  public String getRpcConnnectionURI() throws UnknownHostException {
```

Comment:
nit, return a URI?

Reply:
In the places where we use this method, we always use it as a String(for example, storing it in replication peer config), so returning URI requires the upper layer to call a toString to something else. Since this is just for testing, I made it returning String instead of URI.

Reply:
okay.